platform/darwin/src/MGLGeometry.mm

```diff
@@ -106,3 +106,16 @@ CGPoint MGLPointRounded(CGPoint point) {
 #endif
     return CGPointMake(round(point.x * scaleFactor) / scaleFactor, round(point.y * scaleFactor) / scaleFactor);
 }
+
+MGLMapPoint MGLMapPointMake(mbgl::Point<double> coord, double zoomLevel) {
+    MGLMapPoint point;
+    point.x = coord.x;
+    point.y = coord.y;
```

Comment:
The screen origin on macOS is at the bottom-left, while the origin on iOS is at the top-left. Which is the case here? Perhaps that should be documented as part of `MGLMapPoint.y`.